_layouts/media.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Media
+---          
+        <div class="callout large primary" id="bg-image-media" style="margin-bottom: 0rem;">
+            <div class="row column text-center">
+                <h1>Media</h1>
+                <p>[ Blurb about the December Training material (ppt, pdf, word docs, templates and videos) ]</p>
```

Comment:
"These materials are available for you to use and adapt for your own training purposes"